packages/api/src/endpoints/eventSub/HelixEventSubConduitShard.ts

```diff
@@ -0,0 +1,45 @@
+// import { Enumerable } from '@d-fischer/shared-utils';
+import { DataObject, rawDataSymbol, rtfm } from '@twurple/common';
+// import type { BaseApiClient } from '../../client/BaseApiClient';
+import {
+	type HelixEventSubConduitShardData,
+	type HelixEventSubSubscriptionStatus,
+	type HelixEventSubTransportData,
+} from '../../interfaces/endpoints/eventSub.external';
+
+/**
+ * Represents an EventSub conduit shard.
+ */
+@rtfm('api', 'HelixEventSubConduitShard')
+export class HelixEventSubConduitShard extends DataObject<HelixEventSubConduitShardData> {
+	/**
+	 * The ID of the shard.
+	 */
+	get id(): string {
+		return this[rawDataSymbol].id;
+	}
+
+	/**
+	 * The status of the shard.
+	 */
+	get status(): HelixEventSubSubscriptionStatus {
+		return this[rawDataSymbol].status;
+	}
+
+	/**
+	 * The transport method of the shard.
+	 */
+	get transportMethod(): string {
+		return this[rawDataSymbol].transport.method;
+	}
+
+	/** @private */
+	get _transport(): HelixEventSubTransportData {
+		return this[rawDataSymbol].transport;
+	}
+
+	/** @private */
+	set _status(status: HelixEventSubSubscriptionStatus) {
+		this[rawDataSymbol].status = status;
+	}
```

Comment:
What are these for?

Reply:
I copied them over from HelixEventSubSubscription.ts. Nothing currently is using them for conduits iirc.

Reply:
No need to have private properties that are not used for anything then.